ACID/acid_tbc.sql

```diff
@@ -31324,6 +31324,9 @@ INSERT INTO `creature_ai_scripts` (`id`,`creature_id`,`event_type`,`event_invers
 -- Amani Crocolisk
 ('2404701','24047','0','0','100','3','5000','5000','13000','13000','11','43352','1','0','0','0','0','0','0','0','0','0','Amani Crocolisk - Cast Tail Swip'),
 ('2404702','24047','0','0','100','3','1000','1000','24000','24000','11','43353','1','0','0','0','0','0','0','0','0','0','Amani Crocolisk - Cast Infected Bite'),
+-- Amani'shi Beast Tamer
+('2405901','24059','0','0','100','3','10000','20000','3000','5000','11','43361','4','0','0','0','0','0','0','0','0','0','Amani\'shi Beast Tamer - Cast Domesticate'),
```

Comment:
3-5s repeat timer seems pretty fast for a mind control ability. Should it really be this low?

Reply:
thats the repeattime i found in the sniff, befor i had set it to a higher value with a shorter first timer.